pkg/internal/codegen/parse/util.go

```diff
@@ -395,3 +425,45 @@ func parseScaleParams(t *types.Type) (map[string]string, error) {
 	}
 	return nil, fmt.Errorf(jsonPathError)
 }
+
+// ready subresource requires name,type,JSONPath fields and rest of the field are optional
```

Comment:
We don't need name like `ready` anymore because we decided to use one-line specification for each print column. In the linked issue, `ready` was used to group specifications for a column because column specifications were expressed in multiple lines. 
So lets remove the requirement for additional name like `ready`.

So annotation should read as:
```
// +kubebuilder:printcolumn:name=<name1>,type=<type>,description=<desc>,JSONPath:<.spec.Name>,priority=<0|1>,format=<format>
// +kubebuilder:printcolumn:name=<name2>,type=<type>,description=<desc>,JSONPath:<.spec.Name>,priority=<0|1>,format=<format>
```

Reply:
Ack, I will make the necessary changes.